server/src/email_routes.rs

```diff
@@ -139,10 +139,9 @@ pub fn setemail(req: &mut Request, config: &Config) -> IronResult<Response> {
                         Some(_) => "https",
                         None => "http",
                     };
-                    let full_link = format!("{}://{}:{}/confirmemail?s={}",
+                    let full_link = format!("{}://api.{}/verifyemail?s={}",
                                             scheme,
                                             config.options.general.domain,
-                                            config.options.general.port,
```

Comment:
That change looks unrelated to this PR, but in any case: what happens if you don't run on port 80 or 443?

Reply:
We changed the architecture and now we have an nginx proxy distributing the requests to api.* and for pagekite to allow all of them to come through port 80 or 443